qlib/rl/entries/test.py

```diff
@@ -26,8 +25,8 @@ def backtest(
     action_interpreter: ActionInterpreter,
     initial_states: Sequence[InitialStateType],
     policy: BasePolicy,
-    logger: LogWriter | list[LogWriter],
-    reward: Reward | None = None,
+    logger: LogWriter | List[LogWriter],
+    reward: Reward = None,
```

Comment:
If the default value is `None`, `reward` will be automatically identified as `Optional[Reward]`, so it is not necessary to explicitly declare `Optional`. But it would be okay to add it if it is our preference.